### DISCUSSION TITLE
Am I able to change the port?

### DISCUSSION THREAD

**Discussion by katsie03** [coder/code-server#3956] [Q&A] [answered]
Hi there, I looked through your docs but found nothing with referencing to changing ports for code-server. Since port 8080 is somehow already within use on a clean vps (http-alt) and killing the task does nothing. Is there a way to change the port or does it have to be 8080? Thank you! 

![image](https://user-images.githubusercontent.com/50741678/129073662-7cdc0653-b2f4-4b49-b150-ae682d5687c3.png)

**Comment by jsjoeio** [comment, marked as answer]
Sounds like we need to add that to the docs! It's not very obvious, but if you run `code-server --help`, you see this flag:
```shell
--bind-addr            Address to bind to in host:port. You can also use $PORT to override the port.
```

If you just want to override the port, you can do `PORT=3000 code-server` which is what I usually do.

**Reply by katsie03** [reply]
Thank you very much for this!

**Reply by katsie03** [reply]
@jsjoeio Is there a way to change it to localhost as well instead of 127.0.0.1? Only because it is a VPS and I have no physical local access to the web address

**Reply by jsjoeio** [reply]
Hmm...that ventures a bit out of my areas of expertise so I'm going to pull in @code-asher to see if he knows

**Reply by katsie03** [reply]
Thanks!

**Reply by code-asher** [reply]
Something like `--bind-addr localhost:8081` should do it although code-server may report it is using 127.0.0.1 anyway since that's usually what it resolves to.
